vhdbuilder/packer/vhd-image-builder-gen2.json

```diff
@@ -192,6 +192,16 @@
       "source": "vhdbuilder/parts/k8s/cloud-init/artifacts/kubelet.service",
       "destination": "/home/packer/kubelet.service"
     },
+    {
+      "type": "file",
+      "source": "vhdbuilder/parts/k8s/cloud-init/artifacts/kubesystem.slice",
+      "destination": "/home/packer/kubesystem.slice"
+    },
+    {
+      "type": "file",
+      "source": "vhdbuilder/parts/k8s/cloud-init/artifacts/docker-slice.conf",
+      "destination": "/home/packer/docker-slice.conf"
+    },
```

Comment:
can you do the same for other files
vhdbuilder/packer/vhd-image-builder.json 
vhdbuilder/packer/vhd-image-builder-sig.json

Reply:
added